content/post/2017-04-29-introduction-of-liechi-s-site.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Introduction of Liechi's site
+author: Liechi Zhang
+date: '2017-04-29'
+slug: introduction-of-liechi-s-site
```

Comment:
I revised the contents and pushed the revised file to the forked repository. I am not familiar with how the revision process goes on in GitHub, I hope I am not causing confusions.